homeassistant/components/zeversolar/strings.json

```
@@ -0,0 +1,20 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "ip_address": "[%key:common::config_flow::data::ip%]"
```

This was replaced by `host` in this PR.

```suggestion
          "host": "[%key:common::config_flow::data::host%]"
```